src/app/components/SearchModal/CurrencyList.tsx

```diff
@@ -33,6 +36,9 @@ function CurrencyRow({
   showCurrencyAmount?: boolean;
   onRemove: () => void;
   account?: string | null;
+  isFocused: boolean;
+  index: number;
+  setFocusedIndex: React.Dispatch<React.SetStateAction<number | undefined>>;
```

Comment:
use `onFocus` instead of `setFocusedIndex`, remove index, and define the function like it.
`onFocus: () => void`